controllers/index.js

```diff
@@ -0,0 +1,11 @@
+// controllers/index.js
+
+(function (controllers) {
+
+    var githubController = require("./githubController");
+    
+    controllers.init = function (app) {
```

Comment:
The fix worked! Now you should be able to approve code, including this once you rebase :-D